scripts/dump-ratings-reading-log.sh

```diff
@@ -0,0 +1,25 @@
+psql -U openlibrary -d openlibrary -c "copy (
+  select
+  concat('OL', bookshelves_books.work_id, 'W') as work_id,
+  case
+    when (bookshelves_books.edition_id is NULL)
+    then NULL
+    else concat('OL', bookshelves_books.edition_id, 'M')
+  end as edition_id,
+  bookshelves.name as name,
+  bookshelves_books.created as created
+  from bookshelves_books
+  join bookshelves on bookshelves_books.bookshelf_id = bookshelves.id
+) to stdout with (format csv, header, delimiter E'\t')" | gzip -c > reading-log.txt.gz
+psql -U openlibrary -d openlibrary -c "copy (
+  select
+  concat('OL', ratings.work_id, 'W') as work_id,
```

Comment:
Don't need the table name here, since we're only using one table in this query.
```suggestion
  concat('OL', work_id, 'W') as work_id,
```